index.html

```diff
@@ -458,6 +466,12 @@ <h3>Claims</h3>
         </figcaption>
       </figure>
 
+      <p>
+If a presentation supports predicates, a claim about birthdate (not true or false, but tied to
+a specific date value) can become the basis of proof that at an arbitrary point in time, the
+age of a subject did or will lie within an arbitrary interval.
```

Comment:
I misread "lie" to mean "will not tell the truth" because my mind was in "verifiable credentials land" where some things have a true/false nature. Can we change the language to "the age of the subject has or will be within the specified interval." ... don't even know if that does it. It's hard to understand what's being said here. 

Maybe: "Predicates may be used in presentations to increase privacy by specifying things in a more coarse grained manner, such as the subject being over the age of 65 without exposing their exact birthday."

That feels like it's easier to read... but again, I question the need to introduce the term "predicate" to the document's vocabulary.